### ISSUE TITLE
Audio glitches

### ISSUE THREAD

**Issue by CMoranoA** [Rikorose/DeepFilterNet#512]
Hi! First of all, thank you for your kind words about this library, much appreciated! I've encountered a similar issue with certain audio files. After applying deep filtering, I've noticed outlier short noises, almost like glitches. Has anyone else experienced this? Thanks!

**Comment by mistune** [comment]
What model were you using? I'm seeing a similar issue with DeepFilterNet3. I'm going to retrain the model on DeepFilterNet2

**Comment by viki347** [comment]
I'm seeing a similar issue with DeepFilterNet2.  How do I need to modify it? I would greatly appreciate it if you could help me！ Here is my configuration file：
[train]
seed = 42
device = 
model = deepfilternet2
mask_only = False
df_only = False
jit = False
batch_size = 96
batch_size_eval = 0
overfit = False
num_workers = 4
max_sample_len_s = 5.0
num_prefetch_batches = 32
global_ds_sampling_f = 1.0
dataloader_snrs = -5,0,5,10,20,40
dataloader_gains = -6,0,6
batch_size_scheduling = 0/8, 1/16, 2/24, 4/32, 9/64, 19/96
max_epochs = 50
validation_criteria = loss
validation_criteria_rule = min
early_stopping_patience = 5
start_eval = False

[df]
sr = 48000
fft_size = 960
hop_size = 480
nb_erb = 32
nb_df = 96
norm_tau = 1
lsnr_max = 35
lsnr_min = -15
min_nb_erb_freqs = 2
df_order = 5
df_lookahead = 0
pad_mode = input

[deepfilternet]
conv_lookahead = 0
conv_ch = 16
conv_depthwise = True
convt_depthwise = True
conv_kernel = 1,3
conv_kernel_inp = 3,3
emb_hidden_dim = 256
emb_num_layers = 2
df_hidden_dim = 256
df_gru_skip = none
df_output_layer = linear
df_pathway_kernel_size_t = 1
enc_concat = False
df_num_layers = 3
df_n_iter = 2
gru_type = grouped
gru_groups = 1
linear_groups = 1
group_shuffle = True
dfop_method = real_unfold
mask_pf = False

[distortion]
p_reverb = 0.5
p_bandwidth_ext = 0.0
p_clipping = 0.0
p_zeroing = 0.0
p_air_absorption = 0.5
p_interfer_sp = 0.0

[optim]
lr = 0.0005
momentum = 0
weight_decay = 0.05
optimizer = adamw
lr_min = 1e-06
lr_warmup = 0.0001
warmup_epochs = 3
lr_cycle_mul = 1.0
lr_cycle_decay = 0.5
lr_cycle_epochs = -1
weight_decay_end = -1

[maskloss]
factor = 0
mask = iam
gamma = 0.6
gamma_pred = 0.6
f_under = 2
max_freq = 0

[spectralloss]
factor_magnitude = 0
factor_complex = 0
factor_under = 1
gamma = 1

[multiresspecloss]
factor = 0
factor_complex = 0
gamma = 1
fft_sizes = 512,1024,2048

[sdrloss]
factor = 0

[localsnrloss]
factor = 0.0005

[asrloss]
factor = 0
factor_lm = 0
loss_lm = CrossEntropy
model = base.en

**Comment by viki347** [comment]
I just solved this problem. I'll share it here. Through inspection, I found that my problem was due to incorrect configuration of loss. I estimate that the loss parameters in the config file generated by the author are default and they need to be modified according to the papers. 
Here are some reference links：
[https://github.com/Rikorose/DeepFilterNet/issues/130](url)

**Comment by CMoranoA** [comment]
Hi Viki, can you please expand a little bit your solution? 
Thank you!

**Comment by zxj329** [comment]
> 我也遇到这样的问题，特别是噪声刚刚出现的时刻。

我在-5db的时候，降噪都有比较好的效果
